CodeStarterCamp_Week4/step01.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  step01.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by karen on 2023/03/14.
+//
+
+import Foundation
+
+struct Activity {
+    let name: String
+    let action: (BodyCondition) -> Void
+    
+}
+
+class BodyCondition {
+    var upperBodyStrength: Int //상체근력
+    var lowerBodyMuscleStrength: Int //하체근력
+    var muscularEndurance: Int //근지구력
+    var degreeOfFatigue: Int //피로도
+    
+    func printCondition() {
+        print("현재의 컨디션은 다음과 같습니다.")
+        print("상체 근력 : \(upperBodyStrength)")
+        print("하체 근력 : \(lowerBodyMuscleStrength)")
+        print("근지구력 : \(muscularEndurance)")
+        print("피로도 : \(degreeOfFatigue)")
+    }
+    
+    init(upperBodyStrength: Int, lowerBodyMuscleStrength: Int, muscularEndurance: Int, degreeOfFatigue: Int) {
+        self.upperBodyStrength = upperBodyStrength
+        self.lowerBodyMuscleStrength = lowerBodyMuscleStrength
+        self.muscularEndurance = muscularEndurance
+        self.degreeOfFatigue = degreeOfFatigue
+    }
+    
+}
+
+let sitUp:Activity = Activity(name: "윗몸일으키기", action: {bodyCondition in
```

Comment:
타입 어노테이션은 한칸 띄어 쓴 후 해주는 게 좋습니다!
마찬가지로, 클로저 안에서의 변수 선언 역시 한칸 띄어쓰기 해주시는게 좋아요!

```suggestion
let sitUp: Activity = Activity(name: "윗몸일으키기", action: { bodyCondition in
```

Reply:
오옷 이런 부분도 타입 선언해주는것과 같이 생각해서 띄어주는 연습을 해야될 것 같습니다. 수정했습니다.